stablehlo/reference/Ops.cpp

```diff
@@ -111,6 +111,20 @@ Tensor evalSliceOp(const Tensor &operand, const Sizes &startIndices,
                      inferredTypes[0].cast<ShapedType>());
 }
 
+SmallVector<InterpreterValue> evalCallOp(ArrayRef<Tensor> inputs,
+                                         InterpreterFallback *fallback,
+                                         Process *process, Operation *op,
+                                         StringRef funcName) {
+  SymbolTableCollection symbolTableCollection;
+  auto symbolTable =
+      symbolTableCollection.getSymbolTable(op->getParentOfType<ModuleOp>());
+  auto func = symbolTable.lookupNearestSymbolFrom<func::FuncOp>(
+      op, StringAttr::get(op->getContext(), funcName));
+  SmallVector<InterpreterValue> values = llvm::to_vector(llvm::map_range(
+      inputs, [](const Tensor &t) { return InterpreterValue(t); }));
+  return eval(func.getBody(), values, fallback, process, nullptr);
```

Comment:
Actually wait - do we need any code which adds the result to the current scope?

Reply:
Let's add a single test for this.

Reply:
Discussed offline. Getting the inputs and putting the results in the scope is handled in `eval`.